### PR TITLE
Demo link locator

### PR DESCRIPTION
**Pull request by shaymaa-m**

### CODE REVIEW

File: cypress/e2e/demolinkspage.cy.js
Comment (tayyabjaved1) [inline cypress/e2e/demolinkspage.cy.js:1]: its written correctly, but try not to create a separate `it` function for small cases

File: cypress/e2e/demolinkspage.cy.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+describe('adding all types of selectors', () => {
+   
+
+    it('open page link', () => {
+        cy.visit('https://demoqa.com/links');
+    })
+    
+    it('get element by id', () => {
+
```
Comment (tayyabjaved1) [inline cypress/e2e/demolinkspage.cy.js]: remove space

File: cypress/e2e/demolinkspage.cy.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        cy.get('#simpleLink').contains('Home').should('be.visible')
+    })
+    
+    it('get element by LinkText', () => {
+
```
Comment (tayyabjaved1) [inline cypress/e2e/demolinkspage.cy.js]: remove space

File: cypress/e2e/demolinkspage.cy.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        cy.get('#simpleLink').contains('Home').should('be.visible')
+    })
+    
+    it('get element by LinkText', () => {
+         
+        cy.contains('Forbidden').should('be.visible')
+    })
+
+    it('get element by class', () => {
+
```
Comment (tayyabjaved1) [inline cypress/e2e/demolinkspage.cy.js]: remove space

File: cypress/e2e/demolinkspage.cy.js (hunk 1 above)
Comment (shaymaa-m) [reply cypress/e2e/demolinkspage.cy.js]: done

File: cypress/e2e/demolinkspage.cy.js (hunk 3 above)
Comment (shaymaa-m) [reply cypress/e2e/demolinkspage.cy.js]: done

File: cypress/e2e/demolinkspage.cy.js (hunk 2 above)
Comment (shaymaa-m) [reply cypress/e2e/demolinkspage.cy.js]: done

File: cypress/e2e/demolinkspage.cy.js
Comment (shaymaa-m) [reply cypress/e2e/demolinkspage.cy.js:1]: noted

**Review by tayyabjaved1 — APPROVED** [review]

**Pull request merged**